### PR TITLE
feat:BASE1-3301 Snackbar on Moon Design Tailwind

### PR DESCRIPTION
**Pull request by OladetounJed**
## Checklist

- [ ] You attached screenshots or added description about changes
- [ ] You use [conventional commits naming](https://www.conventionalcommits.org/en/v1.0.0/), e.g. `fix: your changes description [TicketNumber]`, `feat: your feature name [TicketNumber]`
- [ ] You have updated the documentation accordingly.

### CODE REVIEW

**Comment by OladetounJed** [comment]
@yarema184 , here is PR.

The issues are

- Animation doesn't seems to work even after setting the keyframes on the preset file

- setOpen needs improvement

I will refactor and improve the docs once everything is set

File: next-docs/public/examples/snackbarTW/Default.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Snackbar, Button } from '@heathmont/moon-core-tw';
+import { useState } from 'react';
+
+const Example = () => {
+  const [snackbar, setSnackbar] = useState('');
+
+  const openSnackbarHandler = (type: string) => {
```
Comment (karl-kallavus) [inline next-docs/public/examples/snackbarTW/Default.tsx:7]: use React.useCallback

File: next-docs/public/examples/snackbarTW/Default.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        setSnackbar(type);
+      }, 400);
+    } else {
+      setSnackbar(type);
+    }
+  };
+
+  return (
+    <div>
+      <Button onClick={() => openSnackbarHandler('default')}>
```
Comment (karl-kallavus) [inline next-docs/public/examples/snackbarTW/Default.tsx:20]: pass here a callback created by useCallback

File: next-docs/public/examples/snackbarTW/SnackbarPositions.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      }, 400);
+    } else {
+      setSnackbar(type);
+    }
+  };
+
+  return (
+   <div className="flex flex-col lg:flex-row justify-around items-end w-full gap-2">
+     <div>
+      <Button onClick={() => openSnackbarHandler('top-right')}>Top Right Snackbar</Button>
```
Comment (karl-kallavus) [inline next-docs/public/examples/snackbarTW/SnackbarPositions.tsx:21]: same here and below

**Review by karl-kallavus — CHANGES_REQUESTED** [review]

**Pull request merged**